project/profiles/capacity4more/modules/c4m/features/c4m_features_footer/theme/c4m-features-footer-bar-project.tpl.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Template for presenting OG project pages footer.
+ */
+?>
+
+<div class="row">
+  <ul class="c4m-footer-bar clearfix">
+    <li class="first">
+      <a class="logo" href="<?php print $front_page; ?>"
+         title="<?php print t('Home'); ?>">
+        <img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>"/>
+      </a>
+    </li>
+    <li class="c4m-footer-bar-project">
+      <div class="text-uppercase">
+        Explore more projects on
+        <a href="<?php print $project_page; ?>" class="text-orange">Capacity4dev ></a>
+        <br/>
+        <a href=" https://ec.europa.eu/europeaid/home_en" class="">International cooperation and development ></a>
```

Comment:
1. Remove the `class` since it's empty.
2. Remove the extra space from the beginning of the `href`.
3. Let's use `&gt;` instead of `>`.